GUI/UI/UISaveLoad.cpp

```diff
@@ -355,7 +354,8 @@ void MainMenuLoad_EventLoop() {
                 int v26 = param + pSaveListPosition;
                 if (dword_6BE138 == v26) {
                     pMessageQueue_50CBD0->AddGUIMessage(UIMSG_SaveLoadBtn, 0, 0);
-                    pMessageQueue_50CBD0->AddGUIMessage(UIMSG_LoadGame, 0, 0);
+                    // Breaks UI interaction after game load
```

Comment:
Does this fix #158 & #87?

Reply:
Yes, I believe so. And also #76.
Overall this PR should also close #175, #177, #178, #179, #180.